fern/products/docs/pages/integrations/postman.mdx

```diff
@@ -1,14 +1,11 @@
 ---
-title: "Postman Integration"
-description: "Generate a postman collection full of example requests and responses"
+title: "Postman integration"
+description: "Generate a Postman collection full of example requests and responses"
 ---
 
 <Markdown src="/snippets/pro-plan.mdx"/>
 
-You can generate a Postman collection full of example requests and responses and
-publish your collection to Postman. The configuration for the postman generator
-lives in your fern folder, in a file called
-[`generators.yml`](/sdks/overview/project-structure#generatorsyml).
+Postman collection generation is included with Fern Docs. You can generate a Postman collection full of example requests and responses and publish your collection to Postman. The configuration for the Postman generator lives in your fern folder, in a file called [`generators.yml`](/sdks/overview/project-structure#generatorsyml).
```

Comment:
Can we add a sentence or two about what Postman is and why people use it? This assumes everyone knows what Postman is.

Reply:
idea of verbiage:

What is Postman?
[Postman](https://www.getpostman.com/) is an application for easy RESTful API exploration. You can use Postman to test API calls without having to write code. You can save credentials to make it easier to test API calls in various environments, such as sandbox and production.